app/assets/stylesheets/header.scss

```diff
@@ -1,8 +1,16 @@
 .header{
-  border-bottom:solid 2px #5DC9C9;
+  position: fixed;
```

Comment:
Properties should be ordered background-image, border-bottom, border-radius, box-shadow, left, position, top, width